#topic param:
odometry_topic: /vehicle_0/lidar_slam/odom
cloud_topic: /vehicle_0_pcl_render_node/cloud    #######################

vehicle_type: car

global_planning/w_vdir: 1.2
global_planning/w_yawdir: 0.8
global_planning/w_firstnode: 0.2
#viz
view_graph: false
FrontierManager/view_cluster: false
FrontierManager/view_frt: true
# graph
bubble_topo/min_x: 0.6
bubble_topo/min_y: 0.6
bubble_topo/min_z: 0.2
bubble_topo/bubble_min_radius: 0.61
bubble_topo/init_region_size_x: 2.5
bubble_topo/init_region_size_y: 2.5
bubble_topo/init_region_size_z: 2.0
bubble_topo/cube_discrete_size: 0.25
bubble_topo/odom_node_distance: 3.0
bubble_astar/safe_distance: 0.6
# map param
max_update_region_num: 99999999
frontier/candidate_dphi: 0.26
frontier/candidate_znum: 3
lidar_perception/fov_up: 52.0
lidar_perception/fov_down: -7.0
lidar_perception/lidar_pitch: 40.0
lidar_perception/fov_viewpoint_up: 48.0
lidar_perception/fov_viewpoint_down: -5.0
lidar_perception/max_ray_length: 18
parallel_astar/update_connection_timeout: 0.002
parallel_astar/insert_node_timeout: 0.002
FrontierManager/cell_size: 0.4
FrontierManager/noise_cell_range: 1
FrontierManager/good_observation_direction_score: 0.8
FrontierManager/good_observation_force_trust_length: 4.0
FrontierManager/good_observation_trust_length: 12.0
FrontierManager/update_length: 15.0 # 比trust_length长一点就行，多了没用
FrontierManager/cluster_max_size: 7.5
FrontierManager/cluster_min_size: 2.0
FrontierManager/cluster_min_radius: 1.0 # DBSCAN 判断邻居的阈值
FrontierManager/cluster_direction_radius: 0.15 # cos(theta) > cluster_direction_radius
FrontierManager/cluster_minmum_point_num: 3
# 柱形采样
ViewpointManager/sample_pillar_min_height: 0.7
ViewpointManager/sample_pillar_max_height: 0.8
ViewpointManager/sample_pillar_height_layer_num: 4
ViewpointManager/sample_pillar_min_radius: 1.5
ViewpointManager/sample_pillar_max_radius: 3.0
ViewpointManager/sample_pillar_radius_layer_num: 3
ViewpointManager/sample_pillar_circle_sample_num: 8
ViewpointManager/local_tsp_size: 20
# area
box_num: 1
box_0/down: [-5.0, -40.0, -0.2]
box_0/up: [132.0, 75.2, 2.8]
dead_area_num: 0
# trajectory param
max_traj_len: 15.0
DilateRadiusSoft: 0.3 # minco threshold
DilateRadiusHard: 0.25 # collision check threshold
MaxCorridorSize: 4.5
MaxVelMag: 2.0
maxBdrMag: 10000.0
MaxTiltAngle: 0.0 # 最大飞行姿态角度 60°:1.0472 45°:0.7854 30°:0.5236 10°:0.1745
MinThrust: 10.0
MaxThrust: 15.0
VehicleMass: 3.0
GravAcc: 9.8
HorizDrag: 0.70
VertDrag: 0.80
ParasDrag: 0.01
SpeedEps: 0.0001
WeightT: 2000.0
WeightSafeT: 400.0

MaxOmgMag: 2.0
MaxDelta: 0.65
MaxCurvature: 0.4
MaxAccelMag: 2.0
#yjz修改  添加地面车辆相关参数  2025.12.17
# [pos_weight, vel_weight, omg_weight, delta_weight, curvature_weight, acc_weight]
ChiVec: [2.0e+5, 1.5e+5, 5.0e+5, 2.5e+5, 5.0e+5, 2.0e+5]
SmoothingEps: 0.03
IntegralIntervs: 32
RelCostTol: 1.0e-6
#param for yaw opt
yaw_max_vel: 0.5
yaw_rho_vis: 10000.0
yaw_time_fwd: 3.0


fsm/replan_time: 1.0                          # 从 0.5 改为 1.0（规划保护时间窗口）
fsm/replan_time_after_traj_start: 1.0         # 从 0.5 改为 2.0（轨迹起始段保护时间更长）
fsm/replan_time_before_traj_end: 0.5         # 从 0.5 改为 1.5（轨迹末尾提前规划时间）
fsm/thresh_replan: -1.0 